.github/workflows/ci-main.yml

```diff
@@ -26,9 +28,12 @@ jobs:
       uses: actions/checkout@v2
     - name: Build image
       run: |
-        IMAGE_TAG="$(git rev-parse HEAD)"
+        IMAGE_TAG="$(git ls-files -s --full-name "tools" | git hash-object --stdin)"
```

Comment:
should this be a script for unification & to enable adding docs explaining why it exists?

Reply:
Moved into a script.